eth2/utils/eth2_testnet_config/src/lib.rs

```diff
@@ -202,6 +202,7 @@ mod tests {
 
     type E = MainnetEthSpec;
 
+    /* TODO: disabled until testnet config is updated for v0.11
```

Comment:
I'm not sure about the longevity of this test. It's tempting to delete it, or have some flag that turns "hardcoded testnet dirs" on/off?

Reply:
Yeah sounds good, I definitely agree with keeping the feature, just thought maybe we should switch it off at certain points between releases, by the logic that code on `master` is only sometimes going to work with the hardcoded config. E.g. once this PR is merged, the code on master won't be useful for syncing with `testnet5`. Or we can just comment hop

Reply:
(I'm happy to comment hop for now)